### PR TITLE
Map for percentage based discounts support [ch33377]

### PR DESCRIPTION
**Pull request by macharmi**

### CODE REVIEW

File: spec/chartmogul/csv/line_items/one_item_spec.rb [hunk 1]
```diff
@@ -54,7 +54,8 @@
           Description
           Transaction\ fee
           Account\ Code
-          Transaction\ fee\ currency
+          Transaction\ currency
```
Comment (macharmi) [inline spec/chartmogul/csv/line_items/one_item_spec.rb:57]: see https://github.com/chartmogul/platform/blob/1ce70c9498cbb4c3f7d993bb3c8f5297a2fd2561/engines/data_ingestion/app/services/data_ingestion/csv_mapping/line_item.rb#L24

Comment (macharmi) [reply spec/chartmogul/csv/line_items/one_item_spec.rb:57]: PS. the CSV header might be confusing. I informed Blake.

Comment (pkopac) [reply spec/chartmogul/csv/line_items/one_item_spec.rb:57]: Isn't that mapping a bug?
The field is called `transaction_fees_currency`... it's not Transaction currency :|

**Review by pkopac — APPROVED** [review]

File: spec/chartmogul/csv/line_items/one_item_spec.rb (hunk 1 above)
Comment (macharmi) [reply spec/chartmogul/csv/line_items/one_item_spec.rb]: Yes. And I fixed it.

**Pull request merged**